pom.xml

```diff
@@ -87,7 +87,7 @@
         <datasketches.memory.version>1.3.0</datasketches.memory.version>
         <derby.version>10.14.2.0</derby.version>
         <dropwizard.metrics.version>4.0.0</dropwizard.metrics.version>
-        <errorprone.version>2.7.1</errorprone.version>
+        <errorprone.version>2.8.0</errorprone.version>
```

Comment:
Also need to update the version in  https://github.com/apache/druid/blob/master/licenses.yaml#L353